cea/demand/airconditioning_model.py

```diff
@@ -9,11 +9,12 @@
     [1] Kämpf, Jérôme Henri
         On the modelling and optimisation of urban energy fluxes
         http://dx.doi.org/10.5075/epfl-thesis-4548
+
```

Comment:
of course. That's the goal. I think that is easier possible now. As there are smaller functions in general.